lib/foreman_google/google_compute_adapter.rb

```diff
@@ -12,6 +12,10 @@ def project_id
 
     # ------ RESOURCES ------
 
+    def instance(zone, instance_name)
```

Comment:
Hmm, to my surprise `compute_resource.find_vm_by_uuid` works with the `id` and also with the `name`.
Maybe we could rename the `instance_name` parameter to the `instance_id_or_name`, what do you think?

Reply:
Or add comment to the function that both parameters can be used

Reply:
Yeah, both works for in the google API (it's similar to the Foreman API :) )